uint32 frame_id
int16 el
int16 eh
int16 cl
int16 ch
